### DISCUSSION TITLE
Using both Layers A & B

### DISCUSSION THREAD

**Discussion by Toneking77** [maartentamboer/X-Touch-Mini-FS2020#98] [Q&A] [answered]
First, thank you for your time and effort in creating this awesome utility.  Greatly appreciated!   

I've searched all over but haven't found an example of how to use both Layers.  If someone can point me in the right direction, I'd really be thankful.

All The Best,

Michael
Baltimore, Maryland  USA

**Comment by jlr91** [comment]
Hi, You can find below an example I've made for the C172 .
![C172](https://user-images.githubusercontent.com/83365849/150495360-21991eb8-37e2-4014-816d-0f9ff7f90bdd.jpg)

**Comment by mmue90** [comment, marked as answer]
Hi,
to use the Layer B, you can use the encoder 9-16 and buttons 9-32 in the config-file.

Here's my example for the C152:
https://github.com/mmue90/X-Touch-Mini-Config/blob/main/Configurations/config_C152.json

**Reply by Toneking77** [reply]
Thank you very much!   Greatly appreciated!
